### PR TITLE
release[okta-angular][okta-react][okta-vue]: Publish new versions

### PR DESCRIPTION
**Pull request by manueltanzi-okta**
## Description

* Publish okta-angular v1.1.0
* Publish okta-react v1.2.0
* Publish okta-vue v1.1.0

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [guidelines](/okta/okta-oidc-js/blob/master/CONTRIBUTING.md#commit)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Adding Tests
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation changes
- [x] Other... Please describe: Release

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

## Reviewers

@robertjd @brettritter-okta @nbarbettini

### CODE REVIEW

File: packages/okta-angular/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+# 1.1.0
+
+### Features
+
+-[](TODO: add after merged) - Adds TokenManager configuration parameters.
```
Comment (manueltanzi-okta) [inline packages/okta-angular/CHANGELOG.md]: Will update after https://github.com/okta/okta-oidc-js/pull/388 is merged

File: packages/okta-react/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+# 1.2.0
+
+### Features
+
+-[](TODO: add after merged) - Adds TokenManager configuration parameters.
```
Comment (manueltanzi-okta) [inline packages/okta-react/CHANGELOG.md]: Will update after https://github.com/okta/okta-oidc-js/pull/388 is merged

File: packages/okta-vue/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,9 @@
+# 1.1.0
+
+### Features
+
+-[](TODO: add after merged) - Adds TokenManager configuration parameters.
```
Comment (manueltanzi-okta) [inline packages/okta-vue/CHANGELOG.md]: Will update after https://github.com/okta/okta-oidc-js/pull/388 is merged

File: packages/okta-angular/CHANGELOG.md (hunk 1 above)
Comment (swiftone) [reply packages/okta-angular/CHANGELOG.md]: Why not update now if you know what will be in there?

Comment (manueltanzi-okta) [reply packages/okta-angular/CHANGELOG.md]: The SHA of the merged commit in master (see previous CHANGELOG entries), which will be available after we merge #388 .

Comment (swiftone) [reply packages/okta-angular/CHANGELOG.md]: oh, d'uh.   I somehow thought this was README.

File: packages/okta-react/README.md [hunk 4]
```diff
@@ -201,6 +201,15 @@ Security is the top-most component of okta-react. This is where most of the conf
   Accepts a callback to make a decision when authentication is required. If this is not supplied, `okta-react` redirects to Okta. This callback will receive `auth` and `history` parameters. This is triggered when:
     1. `auth.login` is called
     2. SecureRoute is accessed without authentication
+- **storage** *(optional)*:
+  Specify the type of storage for tokens. The types are:
+  - [`localStorage`](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage)
+  - [`sessionStorage`](https://developer.mozilla.org/en-US/docs/Web/API/Window/sessionStorage)
+  - [`cookie`](https://developer.mozilla.org/en-US/docs/Web/API/Document/cookie)
+
+  Defaults to `localStorage` and will fall back to `sessionStorage`, and/or `cookie` if no one of the previous type is available.
```
Comment (nbarbettini) [inline packages/okta-react/README.md]: In all the readmes, I suggest changing this to:

> Defaults to `localStorage`. If [local storage](https://developer.mozilla.org/en-US/docs/Web/API/Web_Storage_API/Local_storage) is not available, falls back to `sessionStorage` or `cookie`.

**Pull request merged**